json/mir-common-json/source/mir_common_json.d

```diff
@@ -3,15 +3,38 @@ module mir_common_json;
 import core.stdc.stdio;
 import core.thread.osthread: getpid;
 import mir.format;
+import mir.rc.array;
 import mir.serde;
+import mir.stdio;
+
+static immutable exc = new Exception("Invalid deserialization implementation");
 
 void validate(alias calc)()
 {
     foreach (v; [
         `{"coordinates":[{"x":2.0,"y":0.5,"z":0.25}]}`,
         `{"coordinates":[{"y":0.5,"x":2.0,"z":0.25}]}`])
         if (calc(v) != Coordinate(2.0, 0.5, 0.25))
-            throw new Exception("Invalid deserialization implementation");
+            throw exc;
+}
+
+@nogc:
```

Comment:
Seems like a micro-optimization - I have some doubts that the average D programmer would disable GC in the usual code.